content/docs/examples/blog-posts.md

```diff
@@ -0,0 +1,81 @@
++++
+title = "Blog posts"
+description = "The following blog posts provide detailed examples and use cases. Be aware that
```

Comment:
Suggestion, move the content of the description to line 8, so it is a paragraph above the Kubeflow blog. 

Change the description to be something like:

Detailed examples and use cases